### PR TITLE
Open firmware system list

### PR DESCRIPTION
**Pull request by ahoneybun**
This adds an open firmware system list to the articles and will be used in the battery thresholds article.

### CODE REVIEW

File: _articles/open-firmware-systems.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+- Keyboard
+- Touchpad 
+- Power Button.
+
+Here is a [link](https://github.com/system76/ec) to the source code for our Open EC.
+
+<table>
+<tbody>
+<tr style="height: 23px;">
```
Comment (jackpot51) [inline _articles/open-firmware-systems.md:36]: Is this height style necessary? Would it look okay without it?

Comment (ahoneybun) [reply _articles/open-firmware-systems.md]: Looks like it's not. I removed it in the newest commit.

Comment (jackpot51) [reply _articles/open-firmware-systems.md]: Awesome, easier to read it now

File: _articles/open-firmware-systems.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+<td>Yes</td>
+<td>No</td>
+</tr>
+<tr>
+<td>lemp10</td>
+<td>Yes</td>
+<td>Yes</td>
+</tr>
+<tr>
+<td>lemp9</td>
```
Comment (jackpot51) [inline _articles/open-firmware-systems.md:57]: What order are these being sorted by? I would normally prefer alphabetical order

Comment (ahoneybun) [reply _articles/open-firmware-systems.md:57]: I don't believe there is a real order though I did put the newer versions at the top. I'll correct that in the next commit.

Comment (ahoneybun) [reply _articles/open-firmware-systems.md]: New commit has the table list updated and in markdown instead of HTML.

**Review by jackpot51 — COMMENTED** [review]
addw2 has been shipping with open firmware and open EC for a while. Also oryp6.

File: _articles/open-firmware-systems.md [hunk 3]
```diff
@@ -33,10 +33,12 @@ Here is a [link](https://github.com/system76/ec) to the source code for our Open
 
 | System | Open Firmware | Open EC |
 |:------:|:-------------:|:-------:| 
+| addw2  | Yes           |         |
```
Comment (jackpot51) [inline _articles/open-firmware-systems.md]: addw2 has Open EC

**Comment by jackpot51** [comment]
It might also be worth linking to http://support.system76.com/articles/transition-firmware/. addw2 and darp6 shipped with different firmware than what is now available, so users could manually transition to the open firmware and open EC firmware manually

File: _articles/open-firmware-systems.md (hunk 3 above)
Comment (ahoneybun) [reply _articles/open-firmware-systems.md]: I fixed that in this commit: https://github.com/system76/docs/pull/539/commits/0bf9efbbd6a5ebda153ce5e83332f62d901abde6

**Comment by ahoneybun** [comment]
> It might also be worth linking to http://support.system76.com/articles/transition-firmware/. addw2 and darp6 shipped with different firmware than what is now available, so users could manually transition to the open firmware and open EC firmware manually

Added that in commit https://github.com/system76/docs/pull/539/commits/da70aa76021b3a339a744077e5f0c7b77c820748 .

**Review by jackpot51 — APPROVED** [review]
This looks great to me now!

**Review by jacobgkau — APPROVED** [review]

**Pull request merged**